docs/appendix/intro-video-transcript.md

```diff
@@ -0,0 +1,173 @@
+> their work. A shared set of generic instructions like these make it possible
+> for people to choose between different browsers, but only if the browsers get
+> it right.
+
+A hand places a cutout representing a stack of papers on the top-center of the
+sheet and draws an arrow from that cutout to the cutout of the browser window.
+
+> To do verify their work, the browser maintainers rely on the third part of
+> the web platform: conformance tests.
+
+A hand draws an arrow from the cutout of the scroll to the cutout of the tests.
+
+> We author tests to describe the same behavior as the standards, just
+> formatted in a way that a computer can understand.
+
+A hand draws an arrow from the cutout of the browser window to the cutout of
+the scroll.
+
+> In the process, the maintainers sometimes uncover problems in the design of
+> the specifications and recommend changes to fix them.
+
+A hand draws an arrow from the cutout of the tests to the cutout of the scroll.
+
+> Test authors also find and fix these so-called "spec bugs."
+
+A hand draws an arrow from the cutout of the browser window to the cutout of
+the tests.
+
+> ...and as they implement the standards, the maintainers of each browser
+> frequently write new tests that can be shared with the others.
+>
+> This is how thousands of people coordinate to build the cohesive programming
+> platform that we call the world wide web. The web-platform-tests project is
+> one of the test suites that make this possible.
+>
+> That's pretty abstract, though! Let's take a quick look at the tests
+> themselves.
+
+**A recording of a web browser**
+
+`http://web-platform-tests.live` is entered into the location bar, and the
+browser loads the page.
+
+> The latest version of the tests are publicly hosted in executable form on the
+> web at web-platform-tests.live.
+
+The browser begins scrolling through the enormous list of directories.
+
+> There, were can navigate among all the tests for all the different web
+> technologies.
+>
+> Let's take a look at a typical test.
+
+The browser stops scrolling, and a mouse cursor clicks on `fetch`, then `api`,
+then `headers`, and finally `headers-basic.html`.
+
+> This test is written with WPT's testing framework, testharness.js. The test
+> completes almost instantly, and testharness.js reports that this browser
+> passes all but one subtest. To understand the failure, we can read the source
+> code.
+
+The mouse opens a context menu, selects "View Source", and scrolls to the
+source of the failing test.
+
+> It looks like the failing subtest is for what happens when a `Headers`
+> instance has a custom JavaScript iterator method. That's a strange edge case,
+> but it's important for browsers to agree on every detail!
+
+`https://web-platform-tests.org` is entered into the location bar, and the
+browser loads the page.
+
+> Other types of tests include "reftests" (for verifying the visual rendering
```

Comment:
Maybe move to a reftest on wpt.fyi for visual effect here?

Reply:
Good idea! I'm suggesting more "view source" stuff to demonstrate the relation between test and reference. I'll add some more explanation according to what feels natural in the time it takes to perform this action.